contrib/opentimelineio_contrib/adapters/ale.py

```diff
@@ -116,6 +117,33 @@ def _parse_data_line(line, columns, fps):
                 target_url=source
             )
 
+        # If available, collect cdl values in the same way we do for CMX EDL
+        cdl = {}
+
+        if 'CDL' in metadata:
+            cdl = _cdl_values_from_metadata(metadata['CDL'])
+            if cdl:
+                metadata.pop('CDL')
```

Comment:
It is slightly more pythonic to use `del metadata['CDL']` instead of `pop`

Reply:
Cool! Makes sense since we don't need the returned index